runtime/mrc/src/lib.rs

```diff
@@ -660,17 +674,14 @@ impl_runtime_apis! {
 		}
 	}
 
-	impl cumulus_primitives_core::CollectCollationInfo<Block> for Runtime {
-		fn collect_collation_info(header: &<Block as BlockT>::Header) -> cumulus_primitives_core::CollationInfo {
-			ParachainSystem::collect_collation_info(header)
-		}
-	}
-
-	// TODO should this be here or under a collection of mrc runtime apis?
-	impl pallet_schemas_runtime_api::SchemasRuntimeApi<Block, AccountId> for Runtime {
+	impl pallet_schemas_runtime_api::SchemasRuntimeApi<Block> for Runtime {
 		fn get_latest_schema_id() -> Result<u16, DispatchError> {
 			Schemas::get_latest_schema_id()
 		}
+
+		fn get_by_schema_id(schema_id: SchemaId) -> Result<Option<SchemaResponse>, DispatchError> {
```

Comment:
I think we can just return Option<SchemaResponse>

Reply:
Fixed